src/RoadRegistry.BackOffice/Uploads/ZipArchiveDbaseEntryTranslator.cs

```diff
@@ -10,11 +10,16 @@ public class ZipArchiveDbaseEntryTranslator<TRecord> : IZipArchiveEntryTranslato
         where TRecord : DbaseRecord, new()
     {
         private readonly Encoding _encoding;
+        private readonly DbaseFileHeaderReadBehavior _readBehavior;
         private readonly IZipArchiveDbaseRecordsTranslator<TRecord> _translator;
 
-        public ZipArchiveDbaseEntryTranslator(Encoding encoding, IZipArchiveDbaseRecordsTranslator<TRecord> translator)
+        public ZipArchiveDbaseEntryTranslator(
+            Encoding encoding,
+            DbaseFileHeaderReadBehavior readBehavior,
```

Comment:
The read behavior (that is, to ignore field offsets) was made injectable because some TransactieZones.dbf simply does not have the offsets set correctly. Honestly, I'd rather see the file fixed but out of my hands ATM.